packages/litexa/src/parser/litexa.pegjs

```diff
@@ -370,6 +373,54 @@ DefineStatement "database type definition"
   }
 
 
+AllFileExclusions
+  = results:AllFileExclusionsStatment* {
+    if ( !results ) {
+      return true;
+    }
+    let shouldInclude = true;
+    for ( let i=0; i<results.length; ++i ) {
+      if ( !results[i] ) {
+        shouldInclude = false;
+      }
+    }
+    return shouldInclude;
+  }
+
+AllFileExclusionsStatment
+  = __ EndOfStatementSequence { return true; }
+  / r:ExcludeStatement __ EndOfStatementSequence { return r; }
+  / $(!EndOfStatementSequence .)* EndOfStatementSequence { return true; }
+
+/* litexa [exclude file]
+The `exclude file` statement lets you specify a static [expression](
+#expression) to exclude all the contents of the file. You might want to exclude
+a file of test states from a production build, or exclude a file from versions
+of your skill, as defined by your [DEPLOY](#deploy-variable) variables.
+
+```coffeescript
+exclude file unless DEPLOY.DEBUG
+# excludes a file's contents unless the DEPLOY variable DEBUG was set
+```
+
+Note: this statement only works on files that exclusively have tests or states
+that don't have any transitions to them.
```

Comment:
a couple trivial notes:
1. Upper case "this".
2. Think you could wrap this in a `:::warning`
3. Not sure if it's even worth noting this, since the non-test files this would realistically apply to is probably zero to none?